doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,282 @@
+.. _tutorial_qemu:
+
+Qemu Tutorial
+*************
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+
+   qemu-debugging.rst
+
+
+Tutorial overview
+=================
+
+In this tutorial, we will demonstrate launching an Ubuntu cloud image in a
+virtual machine that uses cloud-init to pre-configure the system during boot.
+
+The goal of this tutorial is to provide a minimal demonstration of cloud-init
+that users can use as a development environment to test cloud-init
+configurations locally prior to launching in the cloud.
+
+
+Why Qemu?
+=========
+
+Qemu is a cross-platform emulator capable of running performant virtual
+machines. Qemu is used at the core of a broad range of production operating
+system deployments and open source software projects (including libvirt, LXD,
+and vagrant) and is capable of running Windows, Linux, and Unix guest operating
+systems. While Qemu is flexibile and feature-rich, we are using it because of
+the broad support it has due to its broad adoption and ability to run on
+\*nix-derived operating systems.
+
+
+What is an IMDS?
+================
+
+Instance Metadata Service is a service provided by most cloud providers
+as a means of providing information to virtual machine instances. This
+service is used by cloud providers to surface information to a virtual machine.
```

Comment:
Not sure if this is my lack of knowledge here - I'm a bit confused by the use of "surface" as a verb. Is this a common terminology that we can assume the users will know/understand?

Reply:
**I** use this term a lot, though this is probably more colloquial than it needs to be. I'll use "to expose" instead.